recipes/openvdb/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "openvdb" %}
+{% set version = "10.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AcademySoftwareFoundation/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 887a3391fbd96b20c77914f4fb3ab4b33d26e5fc479aa036d395def5523c622f
+
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+
+  host:
+    - boost-cpp
+    - blosc
+    - zlib
+    - tbb-devel
+    - jemalloc
+
+  run:
+    - boost-cpp
+    - blosc
+    - zlib
+    - tbb-devel
+    - jemalloc
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/openvdb/openvdb.h  # [unix]
```

Comment:
This doesn't seem to be a header only library, so please test for the other artifacts that you are installing to $PREFIX including a cmake file, a shared library. Also, check that you binary runs in the test phase by calling `vdp_print --help` or something similar.

Reply:
I added more tests in https://github.com/versatran01/staged-recipes/pull/1/commits/646e474ffa3b84ba8bfd4256495f38db99f6f88e but eventually we should run upstreams unit tests as well, right? Or would that waste too much CI time?

In that case, should we write a simple cpp file that includes headers from the lib? I've seen that in [conda-forge/opencv-feedstock. ](https://github.com/conda-forge/opencv-feedstock/).

Reply:
The purpose of the tests in this recipe is to check *packaging*. Unit tests are for upstream to check correctness. Here we only want to check that everything is installed and linked correctly.